lib/index.d.ts

```diff
@@ -129,7 +129,8 @@ export function decryptSessionKey(options: {
     passwords?: string | string[];
 }): Promise<SessionKey | undefined>;
 
-export interface DecryptResultPmcrypto extends DecryptResult {
+export type DecryptResultPmcrypto = Omit<DecryptResult, 'signatures'> & {
+    signatures: (OpenPGPSignature | undefined)[];
```

Comment:
When can one of the elements be undefined?

Reply:
Never, my bad